misc/change_lease_time

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -e
+#
+# Dynamically increase the DHCP lease time, requires killing and restarting
+# dnsmasq as per documentation (SIGHUP does not reload configuration file).
+lease=$1
+if [ -z $lease ]; then
```

Comment:
-z "$lease" -- otherwise it will be a syntax error if it's empty

Reply:
Tested cases where $lease is an empty string or null; both cases worked fine without explicit string interpolation.